src/features/flows/thingy91x/index.ts

```diff
@@ -0,0 +1,315 @@
+                        href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/serial_lte_modem/doc/AT_commands.html',
+                    },
+                ],
+            },
+            {
+                app: 'pc-nrfconnect-cellularmonitor',
+                description: 'Automatically connect and evaluate parameters.',
+            },
+        ],
+    },
+    {
+        ref: 'nRF Cloud multi-service',
+        resources: [
+            {
+                app: 'pc-nrfconnect-serial-terminal',
+                description: 'Serial interface to send commands to the device.',
+                vComIndex: 0,
+                supplementaryLinks: [
+                    {
+                        label: 'Evaluate nRF Cloud services',
+                        href: 'https://docs.nrfcloud.com/GettingStarted.html#next-steps',
+                    },
+                ],
+            },
+            {
+                app: 'pc-nrfconnect-cellularmonitor',
+                description: 'Automatically connect and evaluate parameters.',
+            },
+        ],
+    },
+    {
+        ref: 'Shell Command Line Interface',
+        resources: [
+            {
+                app: 'pc-nrfconnect-serial-terminal',
+                description: 'Serial interface to send commands to the device.',
+                vComIndex: 0,
+                supplemetaryLinks: [
+                    {
+                        label: 'Modem shell commands',
+                        href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/cellular/modem_shell/README.html#overview',
+                    },
+                    {
+                        label: 'AT Commands reference manual',
+                        href: 'https://docs.nordicsemi.com/bundle/ref_at_commands_nrf91x1/page/REF/at_commands/intro_nrf91x1.html',
+                    },
+                ],
+            },
+            {
+                app: 'pc-nrfconnect-cellularmonitor',
+                description: 'Automatically connect and evaluate parameters.',
+            },
+        ],
+    },
+];
+
+const learnConfig = [
+    {
+        label: 'Developer Academy',
+        description: 'Interactive online learning platform for Nordic devices.',
+        link: {
+            label: 'Nordic Developer Academy',
+            href: 'https://academy.nordicsemi.com/',
+        },
+    },
+    {
+        label: 'Best practices',
+        description:
+            'The main aspects and decisions you need to consider before and during the development phase of a low-power cellular Internet of Things product.',
+        link: {
+            label: 'nWP044 - Best practices for cellular IoT development',
+            href: 'https://docs.nordicsemi.com/bundle/nwp_044/page/WP/nwp_044/intro.html',
+        },
+    },
+];
+
+const developConfig = [
+    {
+        ref: 'Prototyping explorer',
+        sampleSource: 'nrf/applications/asset_tracker_v2',
```

Comment:
Ditto. Also: nRF Cloud?

Reply:
The ref is used by the program and not visible to the user.
It has to match the name used in the programConfig.

Not sure what your question is, but the samplesource is wrong because I don't know it yet.

Reply:
Right. It was about the sampleSource.